src/mapping.jl

```diff
@@ -341,18 +351,17 @@ end
 
 function mis_overhead_copylines(ug::MappingGrid{WC}) where {WC}
     sum(ug.lines) do line
-        mis_overhead_copyline(WC <: WeightedMCell ? Weighted() : UnWeighted(), line)
+        mis_overhead_copyline(WC <: WeightedMCell ? Weighted() : UnWeighted(), line, ug.spacing)
     end
 end
 
-function mis_overhead_copyline(w::W, line::CopyLine) where W
-    if W === Weighted
-        s = 4
+function mis_overhead_copyline(w::W, line::CopyLine, s::Int=4) where W
```

Comment:
```suggestion
function mis_overhead_copyline(w::W, line::CopyLine, s::Int) where W
```

Do not use default parameter for low level API.